alembic/versions/20240209100323_82125159e75d_add_rotation_is_shuffled.py

```diff
@@ -0,0 +1,40 @@
+"""add rotation.is_shuffled
+
+Revision ID: 82125159e75d
+Revises: e838b0504d3b
+Create Date: 2024-02-09 10:03:23.935327
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "82125159e75d"
+down_revision = "e838b0504d3b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("rotation", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "is_shuffled",
+                sa.Boolean(),
+                server_default=sa.text("0"),
```

Comment:
```suggestion
                server_default=sa.text("false"),
```
trying to switch to postgres, i think this has to change now :/ will try to figure out systemic fix